tests/knative-serving/knative_serving_test.go

```diff
@@ -173,9 +173,9 @@ func loadKubeConfigOrDie(t *testing.T) *rest.Config {
 	return kubeConfig
 }
 
-func deleteService(servingClient servingtyped.ServiceInterface, name string) {
-	var deleteImmediately int64
-	_ = servingClient.Delete(name, &meta.DeleteOptions{
-		GracePeriodSeconds: &deleteImmediately,
-	})
+func deleteService(t *testing.T, servingClient servingtyped.ServiceInterface, name string) {
+	err := servingClient.Delete(name, &meta.DeleteOptions{})
```

Comment:
Don't forget to call `t.Helper()` before that line, so the error is logged at the correct location is the error trace.